lega/utils/amqp.py

```diff
@@ -65,11 +65,13 @@ def get_connection(domain, blocking=True):
 
 def publish(message, channel, exchange, routing, correlation_id=None):
     """Send a message to the local broker with ``path`` was updated."""
-    LOG.debug(f'Sending to exchange: {exchange} [routing key: {routing}]')
+    correlation_id = correlation_id or get_correlation_id()
+    assert(correlation_id), "You should not publish without a correlation id"
```

Comment:
Won't python assert this "automatically" if it's not given a default value in the parameters? I.e. remove the `=None` in the function declaration parameter list.

Reply:
If `correlation_id` is None, the `or` will trigger the `get_correlation_id()`, the latter could return None, and the assertion barks when this happens, because it should not happen. So basically, I let it fail, and I'll find it in the logs.